localizations/powerbi-visuals-timeline/stringResources/fi-FI/resources.resjson

```diff
@@ -35,12 +36,12 @@
   "Visual_Cells": "Solut",
   "Visual_Cell_SelectedColor": "Valitun solun väri",
   "Visual_Cell_UnselectedColor": "Valitsemattoman solun väri",
-  "Visual_Cell_StrokeColor": "Viivan väri",
-  "Visual_Cell_SelectedStrokeColor": "Valittu viivanleveys",
-  "Visual_Cell_StrokeWidth": "Viivanleveys",
+  "Visual_Cell_SelectedStrokeColor": "Valitun solun viivanleveys",
+  "Visual_Cell_UnselectedStrokeColor": "Valitsemattoman solun viivanleveys",
   "Visual_Cell_EnableManualSizing": "Ota manuaalinen koon muuttaminen käyttöön",
   "Visual_Cell_Width": "Solun leveys",
-  "Visual_Cell_height": "Solun korkeus",
```

Comment:
how's this key written in the code?

Reply:
It depends on you. The localizations fully copied from en-US locale from the dev branch of the visual